src/scripts/addMutation.js

```diff
@@ -0,0 +1,139 @@
+const fs = require('fs');
+const util = require('util');
+fs.access = util.promisify(fs.access);
+fs.readFile = util.promisify(fs.readFile);
+fs.writeFile = util.promisify(fs.writeFile);
+
+if(process.argv.length !== 3) {
+    throw new Error("You must have exactly 1 command line argument.");
+}
+
+let args = process.argv[2].split(".");
+
+if(args.length !== 2) {
+    throw new Error("Argument must be in the format of Entity.methodName");
+}
+
+async function ensureControllerExists(name) {
+    const controllerFilePath = `./src/app/controllers/${name}Controller.js`;
+    const appFilePath = `./src/app/App.js`;
+    try {
+        await fs.access(controllerFilePath);
+    } catch (e) {
+        console.log("Controller does not exist.");
+        await fs.writeFile(controllerFilePath, `class ${name}Controller {
+    constructor(app) {
```

Comment:
This identation though :O

Reply:
?

Reply:
Looks like the ` ` is misaligned on github